js/mail.js

```diff
@@ -587,6 +591,10 @@ var Mail = {
 
 			// focus 'to' field automatically on clicking New message button
 			$('#to').focus();
+			if (!_.isUndefined($(data.currentTarget).data().email)) {
```

Comment:
I get a similar error in Firefox:
TypeError: $(...).data(...) is null          mail.js:594:22